packages/atomic-react/src/components/stencil-generated/index.ts

```diff
@@ -37,6 +37,10 @@ export const AtomicQueryError = /*@__PURE__*/createReactComponent<JSX.AtomicQuer
 export const AtomicQuerySummary = /*@__PURE__*/createReactComponent<JSX.AtomicQuerySummary, HTMLAtomicQuerySummaryElement>('atomic-query-summary');
 export const AtomicRatingFacet = /*@__PURE__*/createReactComponent<JSX.AtomicRatingFacet, HTMLAtomicRatingFacetElement>('atomic-rating-facet');
 export const AtomicRatingRangeFacet = /*@__PURE__*/createReactComponent<JSX.AtomicRatingRangeFacet, HTMLAtomicRatingRangeFacetElement>('atomic-rating-range-facet');
+export const AtomicRecsInterface = /*@__PURE__*/createReactComponent<JSX.AtomicRecsInterface, HTMLAtomicRecsInterfaceElement>('atomic-recs-interface');
+export const AtomicRecsList = /*@__PURE__*/createReactComponent<JSX.AtomicRecsList, HTMLAtomicRecsListElement>('atomic-recs-list');
+export const AtomicRecsResult = /*@__PURE__*/createReactComponent<JSX.AtomicRecsResult, HTMLAtomicRecsResultElement>('atomic-recs-result');
+export const AtomicRecsResultTemplate = /*@__PURE__*/createReactComponent<JSX.AtomicRecsResultTemplate, HTMLAtomicRecsResultTemplateElement>('atomic-recs-result-template');
```

Comment:
This one should be omitted from the output, similar to search interface.

You can do so in `stencil.config.ts`, in the `react` output target.

Reason is that templating is done with a JSX function, not a result template HTML component.